packages/runner-ct/src/plugins/VueDevtools.tsx

```diff
@@ -0,0 +1,51 @@
+import '@lmiller1990/vue-devtools-inline'
```

Comment:
@dmtrKovalenko I decided to publish our fork on my own account for now, so we should be in a position to merge this up.

Reply:
Nice thank you, it is a perfect solution